packages/components/tests/dummy/app/styles/pages/db-gutters.scss

```diff
@@ -0,0 +1,47 @@
+// GUTTERS
+
+// Loose spacing scale, move out to centralize for re-use:
+// (See: https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=2008%3A3)
+:root {
+    --hds-spacing-xxs: 4px;
+    --hds-spacing-xs: 8px;
+    --hds-spacing-sm: 12px;
+    --hds-spacing-md: 16px;
+    --hds-spacing-lg: 24px;
+    --hds-spacing-xl: 36px;
+    --hds-spacing-xxl: 48px;
+}
+
+// TODO: Determine best default spacing size
+$default-spacing: var(--hds-spacing-md); // set as Sass variable to not expose
+
+.hds-gutters {
+    // vertical (make default ?): (Find out most common use cases)
+    // &:not([class*="hds-gutters-horizontal"]), // yes/ no?
+    &-vertical {
+      > * {
+        &:not(li) {display: block;}
```

Comment:
see comment above about the `:not()` selector

plus I am not sure why we need to intervene on the `display` property of the children (potentially impacting the layout of components used as children, see attached video)

https://user-images.githubusercontent.com/686239/186160262-6c469245-ac19-435a-8835-1f36770a22b1.mov

Reply:
I have since updated this to remove the display styles for children, thanks.